frontend/src/metabase/actions/components/ActionViz/ActionVizForm.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from "react";
+import { t } from "ttag";
+
+import type {
+  WritebackQueryAction,
+  WritebackParameter,
+  OnSubmitActionForm,
+  ActionDashboardCard,
+  ParametersForActionExecution,
+  VisualizationSettings,
+  Dashboard,
+} from "metabase-types/api";
+import { getFormTitle } from "metabase/actions/containers/ActionCreator/FormCreator";
```

Comment:
Not sure if it's a big deal, but it feels off that `actions/components` imports something from `actions/containers`